docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  bot:
+    container_name: masaryk-bot
+    image: masaryk-bot
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file:
+      - .bot.env
+    restart: unless-stopped
+    
```

Comment:
missing EOF

Reply:
Resolved in [7000f9a](https://github.com/DiscordFi/MasarykReborn/pull/11/commits/7000f9a68da05c76fe7c75df8575120f25107203)